src/services/apimap-field-builder.js

```diff
@@ -106,6 +106,35 @@ function ApimapFieldBuilder(model, column, options) {
     return validations;
   }
 
+
+  // NOTICE: Remove Sequelize.Utils.Literal wrapper to display actual value in UI.
+  //         Keep only simple values, and hide expressions.
+  //         Do not export literal values to UI by default.
+  function unwrapLiteral(literalValue, columnType) {
+    let value;
+
+    if (_.isString(literalValue)) {
+      if (['true', 'false'].includes(literalValue.toLowerCase())) {
+        value = Boolean(literalValue);
+      } else if (!_.isNaN(_.toNumber(literalValue))) {
+        if (columnType instanceof DataTypes.NUMBER) {
+          value = _.toNumber(literalValue);
+        } else {
+          value = literalValue;
+        }
+        // NOTICE: Only single quotes are widely considered valid to delimitate string values.
+      } else if (literalValue.match(/^'.*'$/)) {
+        value = literalValue.substring(1, literalValue.length - 1);
+      }
+    } else if (_.isBoolean(literalValue)) {
+      value = Boolean(literalValue);
```

Comment:
What does this mean? In pseudo code, I see: "if it is a boolean, then transform to boolean" 😅

Reply:
So true :D